OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -140,6 +140,24 @@ void DisguiseAs(Actor self, Actor target)
 				t.OnEffectiveOwnerChanged(self, oldEffectiveOwner, AsPlayer);
 		}
 
+		public void DisguiseAs(string actorType, Player newOwner)
```

Comment:
This should also have an `Actor self` argument for consistency with everything else.